suite-native/module-connect-device/src/components/ConnectDeviceScreen.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNode } from 'react';
+import { ImageBackground } from 'react-native';
+
+import { Screen } from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+type ConnectDeviceScreenProps = {
+    children: ReactNode;
+};
+
+const contentStyle = prepareNativeStyle(_ => ({
+    flex: 1,
+}));
+
+export const ConnectDeviceScreen = ({ children }: ConnectDeviceScreenProps) => {
```

Comment:
Uff. Good question. It's a wrapper for the Screen.tsx so that we apply the background image. However it is confusing that it has the name screen. I'll add some better suffix

Reply:
[a376ae7](https://github.com/trezor/trezor-suite/pull/9479/commits/a376ae77aaf2859f0631845b07abca2da22cac44)